docs/topics/wasm/wasm-debugging.md

```diff
@@ -37,7 +37,7 @@ Create a project using the Kotlin Multiplatform wizard:
    >
    {style="note"}
 
-2. In **composeApp** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserDevelopmentRun** task.
+2. In **wasmdemo** | **Tasks** | **kotlin browser**, select and run the **wasmJsBrowserDevelopmentRun** task.
```

Comment:
JFYI
the demo project name was changed to `WasmDemo`, but this content still use the old name `composeApp`